tests/hw-accel/amdgpu/basic/tests/basic-test.go

```diff
@@ -116,5 +117,41 @@ var _ = Describe("AMD GPU Basic Tests", Ordered, Label(amdparams.LabelSuite), fu
 						nodeLabellerEnabled, *deviceconfig.GetEnableNodeLabeller(deviceConfigBuilderNew)))
 			}
 		})
+
+		It("Device Plugin", func() {
```

Comment:
I would suggest to synchronize the Describe / Context / It texts to describe the story of what is being tested. This way, at runtime, the user can read the testcase's _story_